code/modules/admin/verbs/adminjump.dm

```diff
@@ -16,7 +16,7 @@
 				continue
 			turfs.Add(T)
 
-	var/turf/T = pick_n_take(turfs)
+	var/turf/T = safepick(turfs)

```

Comment:
It only picks once and throws out the list, pick_n_take() isn't doing much here.